### ISSUE TITLE
Message - This program is blocked by group policy

### ISSUE THREAD

**Issue by imoldfella** [Microsoft/vcpkg#1089]
I'm using visual studio 2017 dev prompt, run as administrator. Windows 10 Pro. CMake 3.8.1
After ever vcpkg command, it shows the message "This program is blocked by group policy. For more information, contact your system administrator." 

E:\dev\vcpkg>.\vcpkg integrate install
Applied user-wide integration for this vcpkg root.

All MSBuild C++ projects can now #include any installed libraries.
Linking will be handled automatically.
Installing new libraries will make them instantly available.

CMake projects should use -DCMAKE_TOOLCHAIN_FILE=E:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake
**This program is blocked by group policy. For more information, contact your system administrator.**

**Comment by JackBoosY** [comment]
Hi @imoldfella , thanks for reporting this issue!
Please open a new issue if this is still a problem for you.
Thx.
